Bio/SeqIO/PdbIO.py

```diff
@@ -255,6 +255,8 @@ def PdbAtomIterator(handle):
     from Bio.File import UndoHandle
     undo_handle = UndoHandle(handle)
     firstline = undo_handle.peekline()
+    if undo_handle.readlines() == []:
```

Comment:
Probably you should use ``if not line:`` here?

Reply:
Yeah I ended up using `if peekline() == ''` and it seems to work perfectly.